test/integration/scheduler/priorities_test.go

```diff
@@ -70,6 +76,108 @@ func initTestSchedulerForPriorityTest(t *testing.T, scorePluginName string) *tes
 	return testCtx
 }
 
+func initTestSchedulerForNodeResourcesTest(t *testing.T) *testutils.TestContext {
+	cfg := configtesting.V1beta3ToInternalWithDefaults(t, v1beta3.KubeSchedulerConfiguration{
+		Profiles: []v1beta3.KubeSchedulerProfile{
+			{
+				SchedulerName: pointer.StringPtr(v1.DefaultSchedulerName),
+			},
+			{
+				SchedulerName: pointer.StringPtr("gpu-binpacking-scheduler"),
+				PluginConfig: []v1beta3.PluginConfig{
+					{
+						Name: noderesources.Name,
+						Args: runtime.RawExtension{Object: &v1beta3.NodeResourcesFitArgs{
+							TypeMeta: metav1.TypeMeta{
```

Comment:
I think the conversion can handle it if you don't include the TypeMeta. Or didn't it?